README.md

```diff
@@ -1 +1,67 @@
-# pointboard
+# Point Board
+- [Development](#Development)
+- [Testing backend](#Testing-backend)
+- [Deployment](#Deployment)
+
+# Development
+
+## Install dependencies
+### Backend dependencies
+Run `go install` from `backend/`
+### Frontend dependencies
+From the project root directory:
+```
+cd frontend
+npm install
+```
+
+## Generate html template
+Within the `frontend/` folder, run
+```shell script
+npm run build
+```
+This will create or update `index.html` and `bundle/bundle.js` inside `dist` (The `dist` folder will be created if it doesn't exist). For more information on `npm run build`, take a look at `package.json` and [webpack](https://webpack.js.org/).
+
+The backend will use `frontend/dist/index.html` as a html template, which it populates using data from the database and serves to the browser.
+
+## Develop frontend
+Within the `frontend/` folder, run
+```shell script
+npm start
+```
+This will starts the frontend in development mode and begin listening on port (`8080` by default). Visit `localhost:8080` if it is not opened in browser yet. :x
+
+Note that this is limited in the sense that the frontend doesn't pull data from the database. Only the backend is able to pull data, populate the template, and serves to the browser. Therefore, the page you will see on `localhost:8080` isn't populated with data. However, this is a convenient way to see the effect of  stylesheets, and some frontend scripts.
+
+## Start the database
+To run a local database, make sure you have `redis` installed. Start the server by running ```redis-server```. This will start the database on port `6379` by default.
+
+### Development database vs. testing database 
+You need to run two different database for development and testing. Specify the database url for each database in  `backend/config.go`.
```

Comment:
Just a random thought: maybe we can allow users to pass in values by [env vars](https://github.com/aws/amazon-ecs-cli-v2/wiki/Developing-With-Environment-Variables) so that they don't need to change the source code and it would facilitate deployment (also it might be a good idea to provide them with the Dockerfiles)

Reply:
Thank you!! Yes that makes sense and would be a great thing to do later when I update this project. And I really like the idea of containerizing frontend separately -  I actually almost did this. It is in fact the kind of structure that I am more familiar withh